R/dots.R

```diff
@@ -97,7 +97,8 @@ dots_clean_empty <- function(dots, is_empty, ignore_empty) {
 
   if (n_dots) {
     which <- match.arg(ignore_empty, c("trailing", "none", "all"))
-    switch(which,
+    switch(
+      which,
```

Comment:
It's the former. The idea is that it is a special case of this style:

```r
list(
  a,
  b
)
```

where the main arguments go onto the first line. There could be one, two or three of them. A rough guide is that anything before `...` is on the first line, but that does not need to be always the case.